katello/katello-backup

```diff
@@ -46,10 +46,16 @@ optparse = OptionParser.new do |opts|
   @dir = ARGV[0].dup
 end
 
+def cleanup
+  FileUtils.rm_rf @dir
+  run_cmd("katello-service start #{EXCLUDED}") unless @options[:online]
```

Comment:
On this katello-service start, I am getting errors like
```
Redirecting to /bin/systemctl start  mongod.service
^[[Cshell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
chdir: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
```

I think this is because the current working directory has been deleted. I think a `cd` has to happen somewhere either before or after removing @dir (haven't tested though)